operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -376,6 +376,16 @@ spec:
 #      service : ""
 #      url: ""
 
+##########
+#  ---
+#  for_display:
+#
+# A list of annotations that Kiali will look for and display, for every workloads and services, in their respective details pages.
+#    ---
+#    annotations:
+#    - display_name: "My Annotation"
```

Comment:
This cr .yaml file is used to document the default settings. This "My Annotation" is not a default (the default is empty list). So change this so it says `annotations: []`. You can put examples in the description text above. So something like:

```
##########
#  ---
#  for_display:
#
# A list of annotations that Kiali will look for and display, for every workload and service, in their respective details
# pages. Each item in the list is a dict that has "name" and "display_name" to indicate which annotation is to be displayed
# and how it should be displayed. For example: 
# - name: "domain/my-annotation"
#   display_name: "My Annotation"
#
#    ---
#    annotations: []
```